server/channels/web/handlers.go

```diff
@@ -317,6 +319,8 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	logUserID = c.AppContext.Session().UserId
```

Comment:
I can move the `c := &Context{...}` further up so that it can be accessed in the defer.  That also reminds me to add that check since there is not always a session.

Reply:
@lieut-data  Does the latest change address your comment?  I think this is a little cleaner.